lib/codegen/fromcto/vocabulary/vocabularyvisitor.js

```diff
@@ -141,6 +145,30 @@ class VocabularyVisitor {
         return null;
     }
 
+    /**
+     * Visitor design pattern
+     * @param {MapDeclaration} mapDeclaration - the object being visited
+     * @param {Object} parameters  - the parameter
+     * @return {Object} the result of visiting or null
+     * @private
+     */
+    visitMapDeclaration(mapDeclaration, parameters) {
+        const mapDeclarationName = mapDeclaration.getName();
+        const mapDeclarationVocabulary = VocabularyManager.englishMissingTermGenerator(null, null, mapDeclarationName);
+
+        parameters.fileWriter.writeLine(0, `  - ${mapDeclarationName}: ${mapDeclarationVocabulary}`);
```

Comment:
Are the whitespaces before these lines standard? i.e., if this is adding to an existing vocab yaml file, will it be at the same indentation level as the one in the yaml file?

Reply:
I don't think we support passing an existing YAML file to this. The purpose of `VocabularyVistitor` is just to bootstrap a new `.voc` file.